### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
Keeping it at 100%

Check it please @jordimas @darccio @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -1102,11 +1102,11 @@ msgstr "clica aquí"
 
 #: src/view/com/modals/DeleteAccount.tsx:208
 msgid "Click here for more information on deactivating your account"
-msgstr ""
+msgstr "Clica aquí per a més informació sobre desactivar el teu compte"
 
 #: src/view/com/modals/DeleteAccount.tsx:216
 msgid "Click here for more information."
-msgstr ""
+msgstr "Clica aquí per a més informació"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:1109]: There is a full stop/period missing at the end here

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -4213,11 +4213,11 @@ msgstr "O combina aquestes opcions:"
 
 #: src/screens/Deactivated.tsx:211
 msgid "Or, continue with another account."
-msgstr ""
+msgstr "O continua amb un altre compte."
 
 #: src/screens/Deactivated.tsx:194
 msgid "Or, log into one of your other accounts."
-msgstr ""
+msgstr "O inicia sessió en un altre dels teus comptes"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:4220]: And also here

File: src/locale/locales/ca/messages.po (hunk 1 above)
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:1109]: There is a full stop/period missing at the end here
```suggestion
msgstr "Clica aquí per a més informació."
```

File: src/locale/locales/ca/messages.po (hunk 2 above)
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:4220]: And also here
```suggestion
msgstr "O inicia sessió en un altre dels teus comptes."
```

**Comment by pfrazee** [comment]
How's this one looking, is it ready?

**Comment by ivanbea** [comment]
> How's this one looking, is it ready?

Ready to merge, thanks @pfrazee

**Comment by surfdude29** [comment]
@pfrazee just a little reminder that this one is ready to go :)

**Review by pfrazee — APPROVED** [review]
Oh right on, thanks everyone!

**Pull request merged**